adobe/react-spectrum
# Add ListView text overflow

GH-3052

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Set overflowMode="wrap" in ListView. 
2. Create a long title/de
3. scription/content. 
4. Set a width for the ListView.
5. See that the text wraps the next line
6. Do the same where overflowMode="truncation" or when no overflowMode is set
7. See that the text is cut off and an ellipsis is applied

## 🧢 Your Project:
Quarry (we currently have some custom css to mimic this behavior in Inventory)
